README.md

```diff
@@ -32,42 +10,40 @@ Description of the function/functions.
 - To run unit tests: `make test`
 - To lint the repo: `make lint`
 
-## Running Locally with Docker
+### Running the Lambda Function Locally with Docker
 
-<https://docs.aws.amazon.com/lambda/latest/dg/images-test.html>
+1. Build the container:
 
-- Build the container:
+   ```bash
+   docker build -t alma-sapinvoices-ui-dev:latest .
+   ```
 
-  ```bash
-  docker build -t my_function:latest .
-  ```
+2. Run the default handler for the container:
 
-- Run the default handler for the container:
+   ```shell
+   docker run -e WORKSPACE=dev -p 9000:8080 alma-sapinvoices-ui-dev:latest
+   ```
 
-  ```bash
-  docker run -e WORKSPACE=dev -p 9000:8080 my_function:latest
-  ```
+3. Post to the container:
 
-- Post to the container:
+   ```shell
+   curl -XPOST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{"version": "2.0", "routeKey": "$default", "rawPath": "/", "rawQueryString": "", "headers": {}, "requestContext": {"http": {"method": "GET", "path": "/", "protocol": "HTTP/1.1", "sourceIp": "192.0.2.1", "userAgent": "agent"}}}'
```

Comment:
Kind of subtle, but I think this is **really** nice!  If I'm understanding, this is a minimal form of the payload that a lambda function URL passes to the lambda, and therefore it's sufficient to have the `asgi-wsgi` + flask app respond to a request?

Reply:
Yes but also see comment https://github.com/MITLibraries/alma-sapinvoices-ui/pull/1#discussion_r1659229294.